content/pages/volunteers.md

```diff
@@ -0,0 +1,96 @@
+Title: Volunteer Opportunities
+Category: pages
+Summary: Volunteer opportunities available with WiCS undergraduate.
+
```

Comment:
oh, Elana said you don't even need the category or summary metadata - wanna get rid of those 2 lines?